docs/source/contrib_docs/getting-started.rst

```diff
@@ -1,22 +1,14 @@
-=================================
```

Comment:
Be aware that every time we move/rename files like this, we risk breaking links that other people have made into the docs. As far as I know there's no way to create redirects on RTD. That's not to say we can never rename files, but we should be conservative about it.